### ISSUE TITLE
setDelimiter() does not change the actual delimiter. Comma is used even if it's set to another character

### ISSUE THREAD

**Issue by otzy** [thephpleague/csv#301]
## Issue summary

Writer setDelimiter() method does not actually change delimiter in the saved file. I set it to semicolon, but still comma is used 

### System informations

_(In case of a bug report Please complete the table below)_

| Information | Description |
|--------------|---------|
| League\Csv version | 9.0.0 |
| PHP/HHVM version | 7.2 |
| OS Platform | Windows 10 |

`$this->document->setCsvControl($delimiter, $this->enclosure, $this->escape);` after line 337 in AbstractCsv fixes the issue. But probably it was intended to be done in resetProperties()? But that function is empty.

I guess the same problem will be with setEnclosure and setEscape as well.

## Standalone code, or other way to reproduce the problem
```

        $writer = Writer::createFromPath($file_name);

        $writer->setDelimiter(';');
        $writer->setEscape('\\');
        $writer->setEnclosure('"');
        $writer->insertOne(['a', 'b']);

```
### Expected result
`a;b`
### Actual result
`a,b`

**Comment by nyamsprod** [comment]
@otzy thanks for reporting this issue but this was already reported and fix on version 9.0.1 see #244

**Comment by otzy** [comment]
sorry, my fault. I copied install command from this site https://csv.thephpleague.com/9.0/installation/ and did not check that there are already newer versions released
